R/translate.R

```diff
@@ -80,12 +98,11 @@ deepl_translate <- function(path,
     )
 
     markdown_lines[shortcodes_no] <- sprintf("`%s`", purrr::map_chr(shortcodes, digest::digest))
-    brio::write_lines(markdown_lines, temp_markdown_file)
-    wool <- tinkr::yarn$new(path = temp_markdown_file)
-  } else {
-    wool <- tinkr::yarn$new(path = path)
   }
 
+  brio::write_lines(markdown_lines, temp_markdown_file)
```

Comment:
was this a temporary change for debugging?

Reply:
no, it is intended
as we check for equations as paragraphs ($$ $$) and because there can be or not shortcodes, I'll chose to write the tmp anyway.